src/helpers/github.rs

```diff
@@ -895,6 +1086,8 @@ mod tests {
     #[case("users/notheotherben", GitHubRepoSourceKind::User("notheotherben".into()))]
     #[case("orgs/sierrasoftworks", GitHubRepoSourceKind::Org("sierrasoftworks".into()))]
     #[case("repos/sierrasoftworks/github-backup", GitHubRepoSourceKind::Repo("sierrasoftworks/github-backup".into()))]
+    #[case("gists/abcd", GitHubRepoSourceKind::Gist("abcd".into()))]
+    #[case("gists/starred", GitHubRepoSourceKind::GistStar)]
```

Comment:
Any reason why we're using a different approach here for selecting starred gists relative to what we do for starred repos?

I honestly kinda like this model - though maybe we should just use `"starred"` as the source and allow that to be used for standard `github/repo` cases as well? Would be a bit of an API change but we could presumably grandfather in the legacy `github/star` import type.

Reply:
I am kinda fighting the same battle with myself about this. I think this was due to my unfamiliarity with the codebase when I originally implemented the starred repo backup source